dotnet/DotNetTests/TimeWindowBehavior.cs

```diff
@@ -69,8 +69,31 @@ public void TimeWindowBehavior_Cover_StartShiftedRight()
 
       }
 
-    }
-
+        }
+
+        /// <summary>
+        /// -- original --
+        ///   1/1/1985 10:30:00 AM 0
+        ///   1/2/1985 10:30:00 AM 1
+        ///   1/3/1985 10:30:00 AM 2
+        ///   1/4/1985 10:30:00 AM 3
+        ///   1/5/1985 10:30:00 AM 4
+        ///   1/6/1985 10:30:00 AM 5
+        ///   1/7/1985 10:30:00 AM 6
+        ///   1/8/1985 10:30:00 AM 7
+        ///   1/9/1985 10:30:00 AM 8
```

Comment:
Should these comments stay here or can it be removed?

Reply:
good point - I've updated the comment